frontend/package.json

```diff
@@ -33,18 +33,20 @@
     "test-cypress-ceph": "cd packages/ceph-storage-plugin/integration-tests-cypress && ../../../node_modules/.bin/cypress open --config-file cypress-ceph.json --env openshift=true",
     "test-cypress-ceph-headless": "cd packages/ceph-storage-plugin/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --config-file cypress-ceph.json --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
     "test-cypress-kubevirt": "cd packages/kubevirt-plugin/integration-tests-cypress && ../../../node_modules/.bin/cypress open --env openshift=true",
     "test-cypress-kubevirt-headless": "cd packages/kubevirt-plugin/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
-    "test-cypress-pipelines": "cd packages/pipelines-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
-    "test-cypress-pipelines-headless": "cd packages/pipelines-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
-    "test-cypress-gitops": "cd packages/gitops-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
-    "test-cypress-gitops-headless": "cd packages/gitops-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
-    "test-cypress-knative": "cd packages/knative-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
-    "test-cypress-knative-headless": "cd packages/knative-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
+    "test-cypress-pipelines": "cd packages/pipelines-plugin/integration-tests && yarn run test-cypress",
+    "test-cypress-pipelines-headless": "cd packages/pipelines-plugin/integration-tests && yarn run test-cypress-headless",
+    "test-cypress-gitops": "cd packages/gitops-plugin/integration-tests && yarn run test-cypress",
+    "test-cypress-gitops-headless": "cd packages/gitops-plugin/integration-tests && yarn run test-cypress-headless",
+    "test-cypress-knative": "cd packages/knative-plugin/integration-tests && yarn run test-cypress",
+    "test-cypress-knative-headless": "cd packages/knative-plugin/integration-tests && yarn run test-cypress-headless",
+    "test-cypress-helm": "cd packages/helm-plugin/integration-tests && yarn run test-cypress",
+    "test-cypress-helm-headless": "cd packages/helm-plugin/integration-tests && yarn run test-cypress-headless",
```

Comment:
I don't see updated `packages/helm-plugin/integration-tests/package.json` which includes `yarn run test-cypress` or `yarn run test-cypress-headless`?

Reply:
Updated the packages/helm-plugin/integration-tests/package.json as part of [PR: 8387](https://github.com/openshift/console/pull/8387/files#diff-59473c44254777e9b734378b04568aa3ae19f8cb10e30613240ab8389102a79aR9-R12)

Updated the packages/knative-plugin/integration-tests/package.json as part of [PR: 8315](https://github.com/openshift/console/pull/8315/files#diff-d74e7cb5215a16816b9cbaea03b4bc19f854650d01d1695a6b8a2ae41201cf59R9-R11)

Reply:
Hi @makambalaji, wow so many PRs in flight.  I recommend putting these dependant PRs in this PRs description, and adding a 'WIP' 'do not merge' label until the other PRs have merged.  It's a little concerning that this PR could get 'approved' and merged before the other PRs, which would then break our CI!

Reply:
But I am integrating only pipelines related test cases to CI and it is already merged as part of PR: 8422

Reply:
#8315 has not merged yet so if this PR merges before it, `test-cypress-knative` scripts will not work.  Pls put a WIP until #8315 merges -thanks

Reply:
updated the description